camelot/admin/admin_route.py

```diff
@@ -78,7 +86,7 @@ def action_for(cls, route):
         """
         assert isinstance(route, tuple)
         try:
-            admin = cls._admin_routes.resolve(route)
+            admin = naming_context.resolve(route)
         except KeyError:
```

Comment:
is deze nog een KeyError ?

Reply:
c3fe9e669e3acdd160d853813cb1581dea93c507.